dkroutingtool/src/py/manual_viz.py

```diff
@@ -173,7 +184,9 @@ def main():
         #Add the zone to the zone-to-route tracking map
         zone_route_map[sheet] = []
         #iterate through the df to create routes_for_mapping
-        for index, row in manual_routes.parse(sheet).iterrows():
+        manual_dataframe = manual_routes.parse(sheet)
+        manual_dataframe = manual_dataframe[manual_dataframe['route'] != 'Summary']
```

Comment:
I always find it bug-prone to have logic operations depend on specific string names in a dataframe. At the very least we should make this string a constant. Why do we add this Summary row and then remove it anyhow?

Reply:
This is removed because that's when we read back the modified sheet so we want to ignore the summary data. I am also not so sure that there is a simpler way to identify the rows to ignore.